src/hotspot/share/runtime/frame.cpp

```diff
@@ -1565,7 +1565,7 @@ void FrameValues::validate() {
       prev = fv;
     }
   }
-  // if (error) { tty->cr(); print_on((JavaThread*)nullptr, tty); }
+  // if (error) { tty->cr(); print_on(nullptr, tty); }
```

Comment:
This probably need the same cast as in print_frame_layout, to disambiguate the call.

Reply:
Fixed